test/Libraries/RevitNodesTests/Elements/StructuralFramingTests.cs

```diff
@@ -66,7 +66,16 @@ public void TestLocationOfStructuralFraming()
 
             var beam = e as StructuralFraming;
             Assert.NotNull(beam);
-            var curve = beam.Location;
+            var loc = beam.Location;
+
+            var pt = loc as Point;
+            if (pt != null)
+            {
+                Assert.NotNull(pt);
```

Comment:
well, yes I agree that the `Assert.NotNull(pt)` inside of `pt!=null` is a bit redundant, but at the same time the idea here is that this PR allows for location to return either `Point` or `Curve`. I guess that's what I was thinking here. It makes no sense to account for both in a Unit Test since we are specifically collecting a Beam element, and it will have guaranteed `Curve` Local point. I will remove the point part all together.